### PR TITLE
Revert merge of workqueue branch

### LINKED ISSUE #93: Does anything actually use src/pdm/workqueue/sql/models/*?
Hi Alex,

pylint is getting rather upset about db.Column and many other things in these classes:
https://travis-ci.org/ic-hep/pdm/builds/355493779#L647

But I can't see where there are actually used, perhaps these are old classes which could be removed?

Regards,
Simon

### PR DESCRIPTION
**Pull request by sfayer**
Hi Alex,

I don't think this removes any of your code, please can you check it carefully...

Regards,
Simon

Closes #93 (and increases pylint score to 9.82 :-)

### CODE REVIEW

**Comment by alexanderrichards** [comment]
Oh dear what's happened here? This is not the current workqueue branch, looks like the old one with the asynchronous way of doing things that we abandoned. This branch has already been deleted, I guess Janusz merged with the wrong branch at some point... What a mess :p

File: src/pdm/workqueue/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+#!/usr/bin/env python
```
Comment (alexanderrichards) [inline src/pdm/workqueue/__init__.py]: It shouldn't be necessary to have a hashbang in this file.

Comment (sfayer) [reply src/pdm/workqueue/__init__.py]: Yeah, I know, but pylint can get upset about line endings more easily if the file is empty (I'm not even sure why that file was removed by the revert in the first place).

Comment (alexanderrichards) [reply src/pdm/workqueue/__init__.py]: Are you sure it was removed as the yellow icon on github is for a modification not a new file (which is green). It looks to me like your just adding the hashbang into an existing empty file. Perhaps on the other workqueue branch it had the hashbang already?

**Pull request merged**